build.ps1

```diff
@@ -165,7 +160,7 @@ if(-Not $SkipToolPackageRestore.IsPresent) {
     }
 
     Write-Verbose -Message "Restoring tools from NuGet..."
-    $NuGetOutput = Invoke-Expression "&`"$NUGET_EXE`" install -ExcludeVersion -OutputDirectory `"$TOOLS_DIR`""
+    $NuGetOutput = Invoke-Expression "&`"$NUGET_EXE`" install -ExcludeVersion -PreRelease -OutputDirectory `"$TOOLS_DIR`" -Source https://www.myget.org/F/cake/api/v3/index.json"
```

Comment:
At the minute, there is still a little bit of flux with regard to the 0.3.0 release of Cake.Recipe.  Until that stops, I have been pulling the CI build from the MyGet feed.  Once settled, the use of the MyGet feed can be removed.